x-pack/plugins/siem/public/alerts/components/signals/actions.tsx

```diff
@@ -56,6 +58,7 @@ export const updateSignalStatusAction = async ({
   status,
   setEventsLoading,
   setEventsDeleted,
+  dispatchToaster,
```

Comment:
An alternative approach here would be to pass down `onSuccess`/`onFailure` callbacks that could be managed by the component using these actions. This would hoist the toaster logic out of the actions and keep them more pure to their specific use case. This would also make it easy to disable any success/failure operations by just sending down a `noop`, whereas providing a `noop` `dispatchToaster` here would cause a failure since there's an explicit call to `displaySuccessToast` which requires it.